res/css/views/dialogs/security/_AccessSecretStorageDialog.scss

```diff
@@ -14,109 +14,139 @@ See the License for the specific language governing permissions and
-        color: $alert;
+    .mx_AccessSecretStorageDialog_primaryContainer {
+        .mx_AccessSecretStorageDialog_passPhraseInput {
+            width: 300px;
+            border: 1px solid $accent;
+            border-radius: 5px;
+        }
+
+        .mx_AccessSecretStorageDialog_keyStatus {
+            height: 30px;
+        }
+
+        .mx_AccessSecretStorageDialog_recoveryKeyEntry {
+            display: flex;
+            align-items: center;
+
+            .mx_AccessSecretStorageDialog_recoveryKeyEntry_textInput {
+                flex-grow: 1;
+            }
+
+            .mx_AccessSecretStorageDialog_recoveryKeyEntry_entryControlSeparatorText {
+                margin: 16px;
+            }
+
+            .mx_AccessSecretStorageDialog_recoveryKeyEntry_fileInput {
+                display: none;
+            }
+        }
+
+        .mx_AccessSecretStorageDialog_recoveryKeyFeedback {
+            &::before {
+                content: "";
+                display: inline-block;
+                vertical-align: bottom;
+                width: 20px;
+                height: 20px;
+                mask-repeat: no-repeat;
+                mask-position: center;
+                mask-size: 20px;
+                margin-right: 5px;
+            }
+
+            &.mx_AccessSecretStorageDialog_recoveryKeyFeedback_valid {
+                color: $accent;
+
+                &::before {
+                    mask-image: url('$(res)/img/feather-customised/check.svg');
+                    background-color: $accent;
+                }
+            }
+
+            &.mx_AccessSecretStorageDialog_recoveryKeyFeedback_invalid {
+                color: $alert;
+
+                &::before {
+                    mask-image: url('$(res)/img/feather-customised/x.svg');
+                    background-color: $alert;
+                }
+            }
+        }
+
+        .mx_Dialog_buttons {
+            text-align: initial;
+            display: flex;
+            flex-flow: column;
+            gap: 14px;
+
+            .mx_Dialog_buttons_additive {
+                float: none;
+
+                .mx_AccessSecretStorageDialog_reset {
+                    position: relative;
+                    padding-left: 24px; // 16px icon + 8px padding
+
+                    &::before {
+                        content: "";
+                        display: inline-block;
+                        position: absolute;
+                        height: 16px;
+                        width: 16px;
+                        left: 0;
+                        top: 2px; // alignment
+                        background-image: url("$(res)/img/element-icons/warning-badge.svg");
+                        background-size: contain;
+                    }
+
+                    .mx_AccessSecretStorageDialog_reset_link {
+                        color: $alert;
+                    }
+                }
+            }
+
+            .mx_Dialog_buttons_items {
```

Comment:
This might be more accurately named something like `mx_Dialog_buttons_row`

Reply:
Addressed: https://github.com/matrix-org/matrix-react-sdk/pull/8160/commits/f849bf9dc2d58818dd8d4250c912e806db100adf